app/views/govuk_publishing_components/components/_image_card.html.erb

```diff
@@ -0,0 +1,56 @@
+<%
+  href ||= false
+  image_src ||= false
```

Comment:
Is `image_src` something we use elsewhere, or could this be `image_url`?

Reply:
I was following the convention of the attribute name, i.e. `src=`.